app/reports/edit/template.hbs

```diff
@@ -11,16 +11,11 @@
           disablePatientLink=model.isNew
         }}
       {{/if}}
-      {{#if model.visit.patient.displayPatientId}}
-        <div class="ps-info-group patient-name print-hide">
-          <label class="ps-info-label">{{t 'reports.form.patientId' }}</label>
-          <input class="form-control" value={{model.visit.patient.displayPatientId}}>
-        </div>
-      {{/if}}
+
       {{#if model.visit.startDate}}
         <div class="ps-info-group">
           <label class="ps-info-label">{{t 'reports.form.visitDate' }}</label>
-          <input class="form-control" value={{date-format model.visit.startDate format="DD/MM/YYYY hh:mm a"}}>
+          <input class="form-control" disabled value={{date-format model.visit.startDate format="DD/MM/YYYY hh:mm a"}}>
```

Comment:
The date field should be displayed just as text, not an input element. A tech-savvy/malicious user could tweak the disabled property of the input, and change the date.

Kindly implement this as text: <p> or <span>, not <input>